### DISCUSSION TITLE
No longer maintaining stactools

### DISCUSSION THREAD

**Discussion by gadomski** [radiantearth/stac-spec#1331] [STAC Software]
Hey folks, I spent a little bit of time yesterday and today trying to de-bitrot [stactools](https://github.com/stac-utils/stactools), and the juice isn't really feeling worth the squeeze for me. I don't use **stactools** in any of my projects, and personally think that the community is better served by piecing together other libraries to build STAC (e.g. [rio-stac](https://github.com/developmentseed/rio-stac), [antimeridian](https://github.com/gadomski/antimeridian/), [raster-footprint](https://github.com/pjhartzell/raster-footprint)) rather than having to maintain **stactools** _and_ all those other libraries.

I'm going to put a "currently unmaintained" banner on the repo [here](https://github.com/stac-utils/stactools/commit/76de11c44c478da0ac54b524942f9d5ce87d5d1a). If anyone would like to step up and take over maintinance, please open up an [issue on stactools](https://github.com/stac-utils/stactools/issues). Thanks ya'll! 🙇🏼

**Comment by ircwaves** [comment]
@gadomski -- I like the idea of  _a better service to the community_.  But I'm curious if that goes so far as pointing people to those tools, and the [STAC Item Best Practices](https://github.com/radiantearth/stac-spec/blob/master/best-practices.md#item-practices) document?  Or something like stringing together a template?

**Reply by gadomski** [reply]
I think it'd be appropriate to have tooling/pattern guidance in the STAC documentation, or in other places (e.g. [the tutorials](https://stacspec.org/en/tutorials/)). I don't have a coherent vision for STAC documentation and resources, and I know @PowerChell has been putting some thought into the topic so I'll summon her in case she has thoughts.

But in short, yes.
